keepalived-vip/README

```diff
@@ -0,0 +1,2 @@
+# keepalived-vip
+A side car deplyed with nginx-ingress-controller for HA
```

Comment:
Change filename to README.md

Finish the README, including how to build the binary, as well as info like keepalived template format, environment variable, etc.